code/controllers/configuration.dm

```diff
@@ -40,7 +40,7 @@ var/list/gamemode_cache = list()
 	var/feature_object_spell_system = 0 //spawns a spellbook which gives object-type spells instead of verb-type spells for the wizard
 	var/traitor_scaling = 0 			//if amount of traitors scales based on amount of players
 	var/objectives_disabled = 0 			//if objectives are disabled or not
-	var/protect_roles_from_antagonist = 0// If security and such can be traitor/cult/other
+	var/protect_roles_from_antagonist = 1// If security and such can be traitor/cult/other
```

Comment:
Isn't that in the config.txt?

Reply:
Yes, I recommend setting this in the config, but the default should be on.

Reply:
I think that might be off from an earlier setting. Would explain a lot. I'll enable it if I find out it's off.